draft-ietf-cbor-update-8610-grammar.md

```diff
@@ -215,7 +215,7 @@ BCHAR = %x20-26 / %x28-5B / %x5D-7E / NONASCII / SESC / "\'" / CRLF
 PCHAR = %x20-7E / NONASCII
 NONASCII = %xA0-D7FF / %xE000-10FFFD
 ~~~
-{: #e6527-new2 title="Updated ABNF for BCHAR, SCHAR, and PCHAR"
+{: #e6527-new2 title="Update to character-level ABNF in Appendix B of RFC 8610: BCHAR, SCHAR, and PCHAR"
```

Comment:
```suggestion
{: #e6527-new2 title="Update to ABNF in Appendix B of RFC 8610: BCHAR, SCHAR, and PCHAR"
```

Looking through the figures changed in this I found that addition to be more confusing than helpful. (And there is no comment the document received recently I remember this change to address -- mentioning that not because that'd be a strict requirement for that kind of change at this time, but because if there were, there'd be another extra reason for the "character-level" text that I didn't find).